packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -103,6 +104,7 @@ export default compose( [
 			select( 'core/editor' ).getEditorSettings().richEditingEnabled,
 		isTextModeEnabled:
 			select( 'core/edit-post' ).getEditorMode() === 'text',
+		isRTL: I18nManager.isRTL,
```

Comment:
I think that would be good approach according to my previous [comment here](https://github.com/WordPress/gutenberg/pull/21502#issuecomment-611461921) because it is commonly used on web side.

I also saw this check on web side `const isRTL = () => document.documentElement.dir === 'rtl';`

Do you think it is aligned with the `settings.isRTL` ?

Reply:
Right! I missed that comment, sorry about that!

This `isRTL` definition seems to be internal of the component `components/src/date-time/date.js`.

What I think `settings.isRTL` does is to apply the RTL layout depending on the language setting set to the WordPress Interface, independent of the browser language, so it's what is mostly used in the project. I believe we would want to use it too.

An example of usage is on the [Block Mover](https://github.com/WordPress/gutenberg/blob/38244ad6d313937fa8bf81d33c9e7fe53c1ff416/packages/block-editor/src/components/block-mover/index.js#L192)

What I'm not sure is where should we call `setSettings` with the correct `isRTL` value for mobile. Another chance for this is to set it as soon as the stores are configured, together with `setupLocale` on `gutenberg-mobile` repo.

Maybe @Tug will have a better idea about this.

Reply:
We might also pass it from the native apps as an init setting in the same way it is set on web, but I don't think is really needed. 🤔 

I wonder if we could inject this isRTL setting on the initial props when they arrive to RN, before the editor is set up. Sounds a bit hacky though 🤔

Reply:
List component has shared `edit.js` file between web and mobile so having isRTL in `getSettings()` does make sense. However as you said I'm not sure if place where I inject it to provider is the proper one ([see here](https://github.com/WordPress/gutenberg/pull/21502#discussion_r408122839))

@Tug do you have any opinion about this ?